lib/core/test/unit/Cardano/Wallet/Primitive/TypesSpec.hs

```diff
@@ -96,9 +98,11 @@ spec = do
 
     describe "Buildable" $ do
         it "WalletId" $ do
-            let xprv = generateKeyFromSeed (mempty, mempty) mempty
+            let Right phr =
+                    fromText "abcdefghijklmnop" :: Either TextDecodingError (Passphrase "encryption")
+            let xprv = generateKeyFromSeed (coerce phr, mempty) mempty
```

Comment:
Same remark as before, let's not `coerce` between `encryption` and `seed` because they have rather different semantic. Instead, let's define a dummy seed directly from a raw bytestring (cf previous comment)Same remark as before, let's not `coerce` between `encryption` and `seed` because they have rather different semantic. Instead, let's define a dummy seed directly from

Reply:
Fixed in  https://github.com/input-output-hk/cardano-wallet/pull/381/commits/fc2b3c898714e6ce5c4b5117a9938fe538a05e85